app/Http/Transformers/EventTransformer.php

```diff
@@ -15,18 +15,22 @@ class EventTransformer extends TransformerAbstract
      */
     public function transform(Event $event)
     {
+        $content = [];
+        $contentString = substr($event->content, 1, -1);
+        $contentStringWithNoQuotations = str_replace('"', '', $contentString);
+        $contentArray = explode(',', $contentStringWithNoQuotations);
+
+        foreach ($contentArray as $attribute) {
+            $attributeParts = explode(':', $attribute);
+            $content[$attributeParts[0]] = $attributeParts[1];
+        }
```

Comment:
Could we use Attribute Casting to turn this JSON field into an array? https://laravel.com/docs/5.5/eloquent-mutators#array-and-json-casting

Reply:
omg yes thank you @sbsmith86 ! This is so much better I didn't know about this!!

Reply:
Nice, glad this worked! 👍